package.json

```diff
@@ -35,12 +35,14 @@
     }
   },
   "scripts": {
+    "a11y": "node scripts/a11y/index.js",
```

Comment:
This file shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.